docs/Guilding/meme-driven-organizations.md

```diff
@@ -0,0 +1,92 @@
+
+---
+title: 👻 Meme Driven Organizations
+--- 
+
+> _"The most powerful person in the world is the storyteller" - Steve Jobs_
+
+And if the most powerful person in the world is the storyteller, the most powerful _thing_ in the world is a meme. 🤷‍♂️
+
+First, stop thinking of memes as “funny viral images” and start taking them [almost as seriously as genes](https://en.wikipedia.org/wiki/Memetics). Memes are contagious ideas, captioned images are simply most effective at containing them. As they say, a picture is worth a thousand words.
+
+Memes have driven people to crusades in the case of religions as well as to build new worlds as in the case of U.S.’ founding fathers.
+
+Not all memes are about doing good, but most successful memes are. And even in the case of bloodiest of religious wars & the most gruesome of ethnic cleansing - these twisted minds did _somehow_ think they were doing good.
+
+-   Memes are powerful & dangerous, m’kay?
+    
+
+![](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F365ade34-758f-4953-bc50-70f1541b058a_1201x630.png)
+
+## Meme Driven Organizations
+
+When it comes to meme driven organizations, one thing to know is that they don't compete; they aren't here to "win" & their goal is never "to be no.1 in industry X". They are on a mission to solve a specific problem or serve a specific need and every organization doing the same is a potential ally.
+
+Because they are here to _play_ & not to _win_ - meme driven organizations are playing _[infinite](https://en.wikipedia.org/wiki/Finite_and_Infinite_Games)_ [games as opposed to](https://en.wikipedia.org/wiki/Finite_and_Infinite_Games) _[finite](https://en.wikipedia.org/wiki/Finite_and_Infinite_Games)_ [games](https://en.wikipedia.org/wiki/Finite_and_Infinite_Games):
```

Comment:
```suggestion
Because they are here to _play_ & not to _win_ - meme driven organizations are playing [_infinite_ games as opposed to _finite_ ones](https://en.wikipedia.org/wiki/Finite_and_Infinite_Games):
```